From 6d9a0f5c08bbfd46ef998217f819af50fef45acc Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 27 Jul 2020 15:20:16 +0800
Subject: [PATCH] Test case for Library - BaseFakeLibraryLib.uni file

Description: Change the description in the TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.uni.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.uni | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.uni b/TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.uni
index c048e07141..64ea5fe7e4 100644
--- a/TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.uni
+++ b/TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.uni
@@ -14,7 +14,7 @@
 // **/
 
 
-#string STR_MODULE_ABSTRACT             #language en-US "Fake Library Instance for Incremental Build Test(for I/O APIC library class)."
+#string STR_MODULE_ABSTRACT             #language en-US "Fake Library Instance for Incremental Build Test(for I/O APIC library class) for a test."
 
 #string STR_MODULE_DESCRIPTION          #language en-US "Fake Library Instance try to test the influence which brought by all types of source code files' changing (Copy from PcAtChipsetPkg\Library\BaseIoApicLib)."
 
-- 
2.27.0.windows.1

